tests/test_ergomotion.py

```
@@ -312,17 +293,17 @@ async def test_preset_tv(
 
         await coordinator.controller.preset_tv()
 
-        expected_cmd = coordinator.controller._build_command(ErgomotionCommands.PRESET_TV)
+        expected_cmd = coordinator.controller._build_command(KeesonCommands.PRESET_TV)
         first_call = mock_bleak_client.write_gatt_char.call_args_list[0]
         assert first_call[0][1] == expected_cmd
 
     @pytest.mark.parametrize(
         "memory_num,expected_value",
```

Ruff (0.14.11)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)